public/app/plugins/panel/table/TablePanel.tsx

```diff
@@ -50,8 +50,14 @@ export class TablePanel extends Component<Props> {
         override.properties.push({ id: propId, value: width });
       }
     } else {
+      const options = {
+        pattern: fieldDisplayName,
+        frame,
+        series: data.series,
+      };
+
       overrides.push({
-        matcher: { id: matcherId, options: fieldDisplayName },
+        matcher: { id: matcherId, options },
```

Comment:
This won’t work, this is part of a persisted model, we can only store the displayName here , otherwise the whole data frame would be included in the panel
Json that is saved :)

Reply:
Ah, good point!